helmfile/charts/harbor/init-harbor/files/init.sh

```diff
@@ -76,6 +76,32 @@ configure_OIDC() {
 
 configure_GC() {
     echo "Configuring GC"
+
+    if [ "${GC_FORCE_CONFIGURE}" = "false" ]; then
+        res=$(curl -k -X GET -w "%{http_code}" "${ENDPOINT}/system/gc/schedule" \
+            -u admin:"${HARBOR_PASSWORD}")
+
+        # shellcheck disable=SC3057
+        http_code="${res:${#res}-3}"
+
+        if [ "${http_code}" != "200" ]; then
+            echo "Failed to check if GC is configured: ${res}"
+            exit 1
+        fi
+
+        if [ ${#res} -eq 3 ]; then
+            body=""
+        else
+            # shellcheck disable=SC3057
+            body="${res:0:${#res}-3}"
+        fi
+
+        if [ -n "$body" ]; then
+            echo "GC already configured"
+            return
+        fi
```

Comment:
Nit: Couldn't you compact it to something like this?
```suggestion
        if [ ${#res} -ne 3 ]; then
            echo "GC already configured"
            return
        fi
```

Reply:
@robinAwallace I think you forgot to change the `-eq` operator to `-ne`